src/components/MapView/index.tsx

```diff
@@ -150,6 +150,22 @@ function MapView({ classes }: MapViewProps) {
     const hazardLayerId = urlParams.get('hazardLayerId');
     const baselineLayerId = urlParams.get('baselineLayerId');
 
+    /*
+      In case we don't have hazard or baseline layers we will use the default
+      layer provided in the appConfig defined within `prism.json` file.
+     */
+    if (!hazardLayerId && !baselineLayerId) {
+      const defaultLayer = get(appConfig, 'defaultLayer', '');
+      const layer = LayerDefinitions[defaultLayer as LayerKey];
+      if (layer) {
+        const urlLayerKey =
+          layer.type === 'admin_level_data'
+            ? 'baselineLayerId'
+            : 'hazardLayerId';
```

Comment:
these two strings should be constants so we don't lose track of them.

Reply:
Solved

```
    const HAZARD_LAYER_PARAM = 'hazardLayerId';
    const BASELINE_LAYER_PARAM = 'baselineLayerId';
```